tools/build_script_generator/.gdbinit.in

```diff
@@ -0,0 +1,29 @@
+%% for profile in ["release", "debug"]
+%% if "elf." ~ profile in metadata
+define file_{{profile}}
+    file
+    file {{ metadata["elf." ~ profile][0] }}
+end
+%% endif
+%% endfor
+
+# By default, we load the debug file
+file_debug
+layout split
+focus cmd
+set print pretty
+set print asm-demangle on
+set mem inaccessible-by-default off
+
+# Doesn't work correctly since Ctrl-C will kill openocd
+# target extended-remote | openocd -f openocd.cfg -c "gdb_port pipe"
```

Comment:
This doesn't work btw, when pressing `Ctrl-C` in gdb to interrupt the program, the openocd process is killed. 🤦‍♂️ 
@daniel-k solved this problem in RIOT by starting it as a separate process and connecting via port. 😭
https://github.com/RIOT-OS/RIOT/pull/3619

If this actually worked, it would make it a lot easier.